modules/EvseManager/Charger.cpp

```diff
@@ -752,11 +754,22 @@ void Charger::checkSoftOverCurrent() {
 // returns whether power is actually available from EnergyManager
 // i.e. maxCurrent is in valid range
 bool Charger::powerAvailable() {
+    
+    
+    // std::chrono::system_clock::time_point now = maxCurrentValidUntil;
+    // std::time_t now_c = std::chrono::system_clock::to_time_t(now);
+    // std::tm now_tm = *std::localtime(&now_c);
+    // std::stringstream ss;
+    // ss << std::put_time(&now_tm, std::string("UTC: %Y-%m-%d %H:%M:%S").c_str());
+    // EVLOG(warning) << "maxCurrentValidUntil: " << ss.str();
+
     if (maxCurrentValidUntil < std::chrono::system_clock::now()) {
         maxCurrent = 0.;
         signalMaxCurrent(maxCurrent);
+        //EVLOG(warning) << "maxCurrentValidUntil is in the past!";
     }
 
+    // EVLOG(warning) << "getMaxCurrent(): " << getMaxCurrent();
```

Comment:
This is probably not needed anymore

Reply:
Right, that was old debug code. Good catch! Removed.